jha/src/components/enrolment/ReviewTableList.vue

```diff
@@ -761,117 +858,118 @@ export default {
-        });
-      }
       items.push({
-        label: 'City',
+        label: "Street Address 2",
+        value: this.$store.state.enrolmentModule.resAddressLine2,
+      });
+      items.push({
+        label: "Street Address 3",
+        value: this.$store.state.enrolmentModule.resAddressLine3,
+      });
+      items.push({
+        label: "City",
         value: this.$store.state.enrolmentModule.resCity,
       });
       items.push({
-        label: 'Province',
+        label: "Province",
         value: this.$store.state.enrolmentModule.resProvince,
       });
       items.push({
-        label: 'Postal Code',
+        label: "Postal Code",
         value: this.$store.state.enrolmentModule.resPostalCode,
       });
       items.push({
-        label: 'Jurisdiction',
+        label: "Jurisdiction",
         value: this.$store.state.enrolmentModule.resCountry,
       });
       if (!this.$store.state.enrolmentModule.isMailSame) {
         items.push({
-          label: 'Mailing Address:',
-          value: '',
+          label: "Mailing Address:",
+          value: "",
         });
         items.push({
-          label: 'Street Address 1',
+          label: "Street Address 1",
           value: this.$store.state.enrolmentModule.mailAddressLine1,
         });
-        if(this.$store.state.enrolmentModule.mailAddressLine2) {
-          items.push({
-            label: 'Street Address 2',
-            value: this.$store.state.enrolmentModule.mailAddressLine2,
-          });
-        }
-        if (this.$store.state.enrolmentModule.mailAddressLine3) {
-          items.push({
-            label: 'Street Address 3',
-            value: this.$store.state.enrolmentModule.mailAddressLine3,
-          });
-        }
         items.push({
-          label: 'City',
+          label: "Street Address 2",
+          value: this.$store.state.enrolmentModule.mailAddressLine2,
+        });
+        items.push({
+          label: "Street Address 3",
+          value: this.$store.state.enrolmentModule.mailAddressLine3,
+        });
+        items.push({
+          label: "City",
           value: this.$store.state.enrolmentModule.mailCity,
         });
         items.push({
-          label: 'Province',
+          label: "Province",
           value: this.$store.state.enrolmentModule.mailProvince,
         });
         items.push({
-          label: 'Postal Code',
+          label: "Postal Code",
           value: this.$store.state.enrolmentModule.mailPostalCode,
         });
         items.push({
-          label: 'Jurisdiction',
+          label: "Jurisdiction",
           value: this.$store.state.enrolmentModule.mailCountry,
         });
-      }
-      if (this.$store.state.enrolmentModule.phone) {
-          items.push({
-          label: 'Phone',
+        items.push({
+          label: "Phone",
           value: this.$store.state.enrolmentModule.phone,
         });
-      } 
+      }
       return items;
     },
+    getIfHasSpouse() {
```

Comment:
Could you change this to just "hasSpouse"? Would be clearer

Reply:
I second this. Otherwise looks good.